Source/JavaScriptCore/runtime/ScriptExecutable.cpp

```diff
@@ -268,6 +273,17 @@ CodeBlock* ScriptExecutable::newCodeBlockFor(CodeSpecializationKind kind, JSFunc
         RELEASE_ASSERT(kind == CodeForCall);
         RELEASE_ASSERT(!executable->m_codeBlock);
         RELEASE_ASSERT(!function);
+
+        UnlinkedModuleProgramCodeBlock* unlinkedCodeBlock = executable->unlinkedCodeBlock();
+        ParserError error;
+        if (UNLIKELY(!unlinkedCodeBlock))
```

Comment:
Why is this necessary? If the function is `generateOrGetUnlinkedCodeBlock`, we should always call it since it becomes `get`.